pkg/simple/client/logging/options.go

```diff
@@ -42,21 +42,20 @@ func (s *Options) ApplyTo(options *Options) {
 }
 
 func (s *Options) Validate() []error {
-	errs := []error{}
-
+	errs := make([]error, 0)
 	return errs
 }
 
 func (s *Options) AddFlags(fs *pflag.FlagSet, c *Options) {
-	fs.StringVar(&s.Host, "elasticsearch-host", c.Host, ""+
+	fs.StringVar(&s.Host, "logging-elasticsearch-host", c.Host, ""+
```

Comment:
We should be more careful with configuration name change, it will break backward compatibility.

Reply:
https://github.com/kubesphere/kubesphere/blob/master/pkg/simple/client/events/elasticsearch/options.go#L51
![image](https://user-images.githubusercontent.com/53003665/104675734-071fe480-5721-11eb-9b40-ce7a4810bb28.png)

It has a same configuration name in the `events` module, so I changed these configuration name.